genai/example_test.go

```diff
@@ -141,6 +143,34 @@ func ExampleGenerativeModel_CountTokens() {
 	fmt.Println("Num tokens:", resp.TotalTokens)
 }
 
+func ExampleGenerativeModel_JSONSchema() {
+	ctx := context.Background()
+	client, err := genai.NewClient(ctx, option.WithAPIKey(os.Getenv("GEMINI_API_KEY")))
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.Close()
+
+	model := client.GenerativeModel("gemini-1.5-pro")
+	model.ResponseMIMEType = "application/json"
+	model.ResponseSchema = &genai.Schema{
```

Comment:
Nit: add a comment explaining what the example is about - like in this case the expected json response schema